data/zilliqa/assets/zil1gcdkmkuunj0l6hjl48zzye50g397dgslzdf4uk/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Oasis of Amazing Creatures",
+	"symbol": "OAC",
+	"web": "https://sites.google.com/view/oasis-of-amazing-creatures/home",
+	"links": {
+		"Github": "OasisOfAmazing",
+		"Twitter": "OasisOfAmazing",
+		"Web": "https://sites.google.com/view/oasis-of-amazing-creatures/home"
```

Comment:
remove this

Reply:
Thank you so much for your help!

Reply:
Hi, apologies for the inconvenience. I think i get what you mean. I need to upload the logo file to the correct folder too.

1) I have downloaded "Zilliqa" logo.svg from the master and replaced it in my fork.
2) I have uploaded my logo.svg file to the OAC folder.
3) I have deleted the "links: ... " code in the meta.json.

Did i do it correctly? Thanks!

Reply:
Hi!

1) I have uploaded a PNG and removed the SVG.
2) Twitter has been added in lower case too.

Is it ok?